planning/motion_velocity_smoother/include/motion_velocity_smoother/trajectory_utils.hpp

```diff
@@ -53,6 +53,9 @@ std::vector<double> calcTrajectoryIntervalDistance(const TrajectoryPoints & traj
 boost::optional<std::vector<double>> calcTrajectoryCurvatureFrom3Points(
   const TrajectoryPoints & trajectory, const size_t & idx_dist);
 
+boost::optional<std::vector<double>> calcTrajectoryCurvatureFromFrontSteeringAngle(
```

Comment:
I know the other functions are not documented but can you add some docstring for your function ?